src/backend/src/controllers/teams.controllers.ts

```diff
@@ -62,4 +62,15 @@ export default class TeamsController {
       next(error);
     }
   }
+
+  static async deleteTeam(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { teamId } = req.params;
+      const deleter = await getCurrentUser(res);
+      await TeamsService.deleteTeam(deleter, teamId);
+      return res.status(204);
```

Comment:
you should return a message on the response saying 'Successfully deleted team with id ...', or just return the deleted team.